flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlCreateView.java

```diff
@@ -44,6 +47,8 @@
 	private final SqlIdentifier viewName;
 	private final SqlNodeList fieldList;
 	private final SqlNode query;
+
+	@Nullable
 	private final SqlCharStringLiteral comment;
 
 	public SqlCreateView(
```

Comment:
Could you also add `requireNonNull` for the fields should not be null in `SqlCreateView`?

Reply:
Done.